data-api/application.yaml

```diff
@@ -0,0 +1,5 @@
+postgis:
+  schemas:
+    include:
+      - schema: "psc"
+        alias: "psc-alias"
```

Comment:
I am wondering if providing a file in the datadir could have side-effects. Reading the suggested configuration, it won't have a huge impact (apart from aliasing the PSC schema), but still ? 
relying on https://github.com/georchestra/data-api/blob/main/src/services/ogc-features/src/main/resources/application.yml#L74 it will be loaded no matter what, maybe I'd leave it totally commented out ?